src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
```

Comment:
Cards container should have fixed paddings: _50px_ vertically and _40px_ horizontally.